### PR TITLE
[Form] Fix typo, php-doc and argument in form methods

### PR DESCRIPTION
**Pull request by bhavin-nakrani**
| Q | A |
| --- | --- |
| Branch? | "master" |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | no |
| License | MIT |
- Fix php-doc on methods
- Fix typo in variables
- Run php cs-fixer

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Core/CoreExtension.php [hunk 1]
```diff
@@ -36,12 +36,21 @@ class CoreExtension extends AbstractExtension
      */
     private $choiceListFactory;
 
+    /**
+     * CoreExtension constructor.
+     *
+     * @param PropertyAccessorInterface|null  $propertyAccessor
+     * @param ChoiceListFactoryInterface|null $choiceListFactory
+     */
     public function __construct(PropertyAccessorInterface $propertyAccessor = null, ChoiceListFactoryInterface $choiceListFactory = null)
```
Comment (fabpot) [inline src/Symfony/Component/Form/Extension/Core/CoreExtension.php]: We don't add phpdoc blocks when they don't add any new information that cannot be found in the signature. That's the case here, so this should be removed.

File: src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php [hunk 2]
```diff
@@ -22,7 +22,14 @@
  */
 class UploadValidatorExtension extends AbstractTypeExtension
 {
+    /**
+     * @var TranslatorInterface
+     */
     private $translator;
```
Comment (fabpot) [inline src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php]: We don't document private properties and methods.

Comment (bhavin-nakrani) [inline src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php]: Yes it is private so no need php-doc. it is fixed. but i think it is easy for type hint when we write code. By the way is mention in http://symfony.com/doc/current/contributing/code/standards.html#naming-conventions :)

File: src/Symfony/Component/Form/Extension/DataCollector/FormDataCollector.php [hunk 3]
```diff
@@ -75,6 +75,10 @@ public function __construct(FormDataExtractorInterface $dataExtractor)
 
     /**
      * Does nothing. The data is collected during the form event listeners.
+     *
+     * @param Request         $request
+     * @param Response        $response
+     * @param \Exception|null $exception
```
Comment (fabpot) [inline src/Symfony/Component/Form/Extension/DataCollector/FormDataCollector.php]: Please, remove those new phpdocs (and almost every others you added). We don't add phpdocs when it does not add something that cannot be already inferred by the signature.

File: src/Symfony/Component/Form/Extension/Core/CoreExtension.php [hunk 4]
```diff
@@ -36,12 +36,18 @@ class CoreExtension extends AbstractExtension
      */
     private $choiceListFactory;
 
+    /**
+     * CoreExtension constructor.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/CoreExtension.php]: we don't add such trivial doc blocks (no value added)

File: src/Symfony/Component/Form/Extension/Core/EventListener/FixUrlProtocolListener.php [hunk 5]
```diff
@@ -34,6 +37,9 @@ public function __construct($defaultProtocol = 'http')
         $this->defaultProtocol = $defaultProtocol;
     }
 
+    /**
+     * @param FormEvent $event
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/EventListener/FixUrlProtocolListener.php]: pure duplicata of the method signature

**Comment by nicolas-grekas** [comment]
:-1: unless the added doc blocks add some value to the signature of the methods.

File: src/Symfony/Component/Form/ChoiceList/ArrayChoiceList.php [hunk 6]
```diff
@@ -177,12 +177,13 @@ public function getValuesForChoices(array $choices)
...
-     *                                  corresponding values
-     * @param array    $keysByValues    The original keys indexed by the
-     *                                  corresponding values
+     * @param array    $choices          The array to flatten
+     * @param callable $value            The callable for generating choice values
+     * @param array    $choicesByValues  The flattened choices indexed by the
+     *                                   corresponding values
+     * @param array    $keysByValues     The original keys indexed by the
+     *                                   corresponding values
+     * @param array    $structuredValues
```
Comment (fabpot) [inline src/Symfony/Component/Form/ChoiceList/ArrayChoiceList.php]: We need a description here, if not, it does not help that much.

**Comment by fabpot** [comment]
Same for me here, 👎

**Comment by xabbuh** [comment]
Some of the docblocks will actually help IDEs to show proper type information (scalar type constructor arguments for example). The rest should be reverted or filled with information.

**Comment by nicolas-grekas** [comment]
@bhavin-nakrani please submit a filtered PR if you want.

**Pull request closed without merging**